### ISSUE TITLE
Enterprise Advisory Group

### ISSUE THREAD

**Issue by jasnell** [nodejs/TSC#431]
@dshaw and I have been working to pull together a group of Enterprise Node.js adopters into a formal group chartered under the Community Committee for the specific purpose of providing enterprise adopters a channel through which they can provide feedback to the core project around priorities, pain points, etc. The idea is for this group specifically to represent companies that have adopted Node.js, regardless of whether they are paying members of the Foundation or not. The group had it's first meeting today and will be working on pulling together a draft charter proposal for the @nodejs/community-committee to review. I will be participating in the group as a liaison, ensure that the group has an open communications channel with the TSC. This channel will be bidirectional, with me briefing the group on the current status and directions that core is going, and me providing the TSC with the feedback from the advisory group.

I will provide more detail and background and answer any questions the @nodejs/tsc members may have about this group on the next TSC call. As such, I'm tagging this issue `tsc-agenda` so that it's added.

**Comment by ljharb** [comment]
Airbnb would love to participate in/be represented by this group.

**Comment by jasnell** [comment]
Minutes for the first meeting here: https://github.com/nodejs/user-feedback/issues/18

**Comment by jasnell** [comment]
@ljharb ... excellent, I'll add you to the list.

**Comment by mhdawson** [comment]
I would have liked the opportunity to have attended the first meeting.  I had heard something was in the works but did not see any public invite or details ?

**Comment by bnb** [comment]
@jasnell I'd like to ask why this was not surfaced publicly to the CommComm before or at the same time as being surfaced to the TSC, since you're going to be proposing it as a CommComm initiative. Similarly to what @mhdawson said, I didn't see any public invitation or details of the meeting anywhere - and I know of several individuals that would likely have wanted to participate.

**Comment by Trott** [comment]
Putting it back on the TSC agenda because we ran out of time while talking about it. Won't be able to pick it up again until a January meeting though, because James won't be at any of the remaining December meetings. Good conversation so far, though.

**Comment by bnb** [comment]
I'd like to request to observe in the meeting in which this is discussed. Is that a possibility?

**Comment by fhinkel** [comment]
@bnb I think that makes sense! AFAIK we'll talk about that in January when @jasnell is back.

**Comment by mhdawson** [comment]
Agreed, we'll make sure to invite you as an observer.

**Comment by dshaw** [comment]
If we are going to discuss this, I would like to request to be added as an Observer for this meeting since we want to discuss process. Let's bring in the offending party (me).

**Comment by jasnell** [comment]
+1 to adding @dshaw as an observer to the TSC call.

**Comment by Trott** [comment]
At an earlier meeting, we agreed to table this until January when @jasnell is back. It's going to be auto-added to the agenda this week and again next week, but it won't be until the week after that we actually talk about it.

So, @dshaw, make a note in your calendar for January 3, although that meeting is currently for an inconvenient time for @jasnell, so maybe also make a provisional note in your calendar for January 10. Yikes!

**Comment by dshaw** [comment]
@Trott Thanks for the heads-up.

In the interim, I'll work with @bnb and the CommComm to resolve issues there.

**Comment by jasnell** [comment]
Closing this as there's currently nothing further for the TSC to discuss.

**Comment by ljharb** [comment]
@jasnell could you summarized what was discussed (referred to in advance [here](https://github.com/nodejs/TSC/issues/431#issuecomment-352523868))?
